### ISSUE TITLE
UTM UUID is handled as a string instead of binary data

### ISSUE THREAD

**Issue by realflight1** [opendroneid/receiver-android#8]
The [getUasIdAsString](https://github.com/opendroneid/receiver-android/blob/7d327379f0dbb7442efaf12d654bbe90ad5e4a18/Android/app/src/main/java/org/opendroneid/android/data/Identification.java#L84) method of the `Identification` class handles all IDs as strings, but according to the ASTM remote ID standard (under BASIC ID MESSAGE 5.4.5.6) it should be handled as a 128-bit UUID - binary encoded with network byte order.

**Comment by friissoren** [comment]
You are correct. That is something that is not properly handled in the receiver implementation at the moment. Based on the value of [IdTypeEnum](https://github.com/opendroneid/receiver-android/blob/7d327379f0dbb7442efaf12d654bbe90ad5e4a18/Android/app/src/main/java/org/opendroneid/android/data/Identification.java#L44), it should handle the uasId either as ASCII or binary data. If you have a good suggestions on how to correct this, please upload a Pull Request.

**Comment by gabrielcox** [comment]
Here's one method/suggestion (courtesy stack overflow) :

```
import java.nio.ByteBuffer;
import java.util.UUID;
```

    public static UUID getUUIDFromBytes(byte[] bytes) {
        ByteBuffer byteBuffer = ByteBuffer.wrap(bytes);
        Long high = byteBuffer.getLong();
        Long low = byteBuffer.getLong();

        return new UUID(high, low);
    }

This will return a UUID object.  .toString() will return the "common looking" UUID string in the 4-2-2-2-6 format.
